templates/api-blueprint.swig

```diff
@@ -102,6 +106,10 @@ FORMAT: 1A
 ### {{ resource.title|default('Resource') }} [{{ resource.href }}]
 
 {{ resource.description }}
```

Comment:
@danielgtaylor This is still not fixed. I think there would be a few extra blank lines in the generated API Blueprint if we leave this as it is.